spec/class.dd

```diff
@@ -1179,6 +1179,8 @@ void test()
     int i = s; // i == 4
 }
 ---
+        $(P $(GLINK2 attribute, Attribute)s are ignored for $(AliasThis).
```

Comment:
The documentation tester complains that $(AliasThis) is an nonexistent macro. I guess you meant `$(D AliasThis)`